tracer/src/Datadog.Trace/Tagging/AzureServiceBusTags.cs

```diff
@@ -13,7 +13,7 @@ namespace Datadog.Trace.Tagging
 {
     // Re-implement InstrumentationTags so the built-in Activity can copy over SpanKind
     // without being limited by our tags infrastructure
-    internal partial class AzureServiceBusTags : CommonTags
+    internal partial class AzureServiceBusTags : OpenTelemetryTags
```

Comment:
Just wanted to flag this to make sure I'm not doing something wrong here by inheriting from `OpenTelemetryTags` and removing the `[Tag]` from `SpanKind` and removing `MessagingOperation` entirely from this class.

It _seems_ to work fine, but just want to double check.